source/conf.py

```diff
@@ -11,91 +11,93 @@
 # All configuration values have a default; values that aented out
 # serve to show the default.
 
-import sys, os
+import os
+import sys
+from datetime import datetime
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-sys.path.append(os.path.abspath('_themes'))
+sys.path.append(os.path.abspath("_themes"))
 
 # -- General configuration -----------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
-#needs_sphinx = '1.0'
+# needs_sphinx = '1.0'
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = []
 
 # Add any paths that contain templates here, relative to this directory.
-templates_path = ['_templates']
+templates_path = ["_templates"]
 
 # The suffix of source filenames.
-source_suffix = '.rst'
+source_suffix = ".rst"
 
 # The encoding of source files.
-#source_encoding = 'utf-8-sig'
+# source_encoding = 'utf-8-sig'
 
 # The master toctree document.
-master_doc = 'index'
+master_doc = "index"
 
 # General information about the project.
-project = u'Sublime Text Unofficial Documentation'
-copyright = u'2013-2020 The Sublime Text Community'
+project = u"Sublime Text Unofficial Documentation"
+copyright = u"2013-%s The Sublime Text Community" % datetime.now().year

```

Comment:
Good idea, but please use don't ever use `%`-style formatting. `.format` is much more powerful, but instead I would like to move forward to Python 3.6+ anyway, since sphinx supports that, and thus use f-strings.